### PR TITLE
Created Collection Page

### PR DESCRIPTION
**Pull request by bk1405bgl**
created a collection page, where message is shown 'Collection empty'  or 'here is your collection' // 
created a link to show the Collection listed in the API // 
created page, where the api collection is listed by cover thumbnails as links (but not working yet)

[Issue](https://github.com/bk1405bgl/capstone-project/issues/3)

### CODE REVIEW

File: components/DisplayOwnCollection/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    name: "Name des Comics",
+  },
+  {
+    id: "2",
+    name: "Name des Comics",
+  },
+  {
+    id: "3",
+    name: "Name des Comics",
+  }, */
```
Comment (Manu7864z) [inline components/DisplayOwnCollection/index.js:15]: why is this in comments?

File: components/ApiCollection/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        );
+      }
+      return [...prevComicsInfo, { id: id, isInOwnCollection: true }];
+    });
+  }
+
+  if (!data) {
+    return <h1>Loading...</h1>;
+  }
+
```
Comment (Manu7864z) [inline components/ApiCollection/index.js:32]: i get a little confused by state id and id on this page...

**Review by Manu7864z — CHANGES_REQUESTED** [review]
I guess it should be reviewed by one of the coaches, i struggle getting behind the code in file 2. the rest looks good to me :)

File: components/DisplayOwnCollection/index.js (hunk 1 above)
Comment (bk1405bgl) [reply components/DisplayOwnCollection/index.js:15]: this was a test to see, if the data is displayed, when it exists and when the "empty collection" message is displayed, when the collection is empty.

File: components/ApiCollection/index.js (hunk 2 above)
Comment (bk1405bgl) [reply components/ApiCollection/index.js:32]: me too

File: components/DisplayOwnCollection/index.js (hunk 1 above)
Comment (Kheirah) [inline components/DisplayOwnCollection/index.js:15]: Remove lines of code which are commented out.

File: package.json [hunk 3]
```diff
@@ -12,11 +12,14 @@
   "dependencies": {
     "eslint": "8.35.0",
     "eslint-config-next": "13.2.3",
+    "fetcher": "^0.3.0",
     "next": "13.2.3",
```
Comment (Kheirah) [inline package.json:16]: You don't need the package `fetcher`.

**Review by Kheirah — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (bk1405bgl) [reply package.json:16]: removed

File: components/DisplayOwnCollection/index.js (hunk 1 above)
Comment (bk1405bgl) [reply components/DisplayOwnCollection/index.js]: removed

**Review by stefankorder — APPROVED** [review]
LGTM!

**Pull request merged**